Tests/UnitTests/Networking/Responses/OfferingsDecodingTests.swift

```diff
@@ -48,10 +49,48 @@ class OfferingsDecodingTests: BaseHTTPResponseTest {
     }
 
     func testDecodesSecondOffering() throws {
-        let offering = try XCTUnwrap(self.response.offerings.last)
+        let offering = try XCTUnwrap(self.response.offerings[1])
 
         expect(offering.identifier) == "alternate"
         expect(offering.description) == "alternate offering"
+        expect(offering.metadata) == [:]
+        expect(offering.packages).to(haveCount(1))
+
+        let package = offering.packages[0]
```

Comment:
To avoid the test crashing (instead of failing):
```suggestion
        let package = try XCTUnwrap(offering.packages.onlyElement)
```